src/data_info/services/tickets.clj

```diff
@@ -70,12 +72,14 @@
        :tickets (mapv (partial returnable-ticket-map cm) new-uuids)})))
 
 (defn- remove-tickets
-  [user ticket-ids]
+  [user ticket-ids for-job?]
   (irods/with-jargon-exceptions [cm]
     (validators/user-exists cm user)
     (validators/all-tickets-exist cm user ticket-ids)
     (let [all-paths (mapv #(.getIrodsAbsolutePath (ticket-by-id cm (:username cm) %)) ticket-ids)]
-      (validators/all-paths-writeable cm user all-paths)
+      (if for-job?
+        (validators/all-paths-readable cm user all-paths)
+        (validators/all-paths-writeable cm user all-paths))
```

Comment:
This validation is kind of a kluge because we have no way to verify that a ticket was generated specifically for a job at this time. The only safety measure that we have at the moment is the fact that this query parameter is not going to be exposed in externally facing endpoints. I suppose we could add something to the ticket string indicating that the ticket was created for a job, but that doesn't really seem necessary. If anyone can think of a reason stronger validations should be added (or has a better idea for implementing stronger validations), Please let me know.

Reply:
I agree. Since `for-job` is not publicly exposed, I don't think we need stronger validations at this point (and I can't think of anything better than adding something to the ticket string anyway).